server/routerlicious/packages/services-client/src/historian.ts

```diff
@@ -31,7 +31,8 @@ export const getAuthorizationTokenFromCredentials = (credentials: ICredentials):
  * Implementation of the IHistorian interface that calls out to a REST interface
  */
 export class Historian implements IHistorian {
-    private readonly defaultQueryString: Record<string, unknown> = {};
+    private readonly defaultQueryString: Record<string, string | number | boolean | readonly string[] |
+        readonly number[] | readonly boolean[] | undefined | null> = {};
```

Comment:
"Record<string, string | number | boolean | ...>" is TypeScript expanding the 'ParsedUrlQueryInput' type alias.

If you 'F12' on 'querystring.stringify(..)' you'll see the definition from `@types/node`:

```ts
function stringify(obj?: ParsedUrlQueryInput, ...): string;
```

...and if you F12 on 'ParsedUrlQueryInput' you'll see the alias definition:

```ts
    interface ParsedUrlQueryInput {
        [key: string]: string | number | boolean | ReadonlyArray<string> | ReadonlyArray<number> | ReadonlyArray<boolean> | undefined | null;
    }
```

...which is equivalent to the Record<..> type.

I'd recommend importing and using 'ParsedUrlQueryInput' over the 'Record<..>' type (mostly for brevity.)